docs/source/features/stablehlo.md

```diff
@@ -64,6 +64,31 @@ print(stablehlo.mlir_module())
 The second to last line we used `jax.ShapedDtypeStruct` to specify the input shape.
 You can also pass a numpy array here.
 
+### Inline some weights in generated stablehlo
+
+Suppose that you want to inline some (or all) of the model's weight
+into the generated StableHLO graph as constant. You can accomplish it by
+exporting a different function that calls your model.
+
+The convention used in `jax.jit` is that, all the input of the `jit`'d python
```

Comment:
Suggest: The convention used in `jax.jit` is all inputs to `jit`ed Python functions are exported as parameters, everything else is inlined as constants.

Reply:
done.